offer-manager-domain/src/main/groovy/life/qbic/business/projects/create/CreateProject.groovy

```diff
@@ -16,22 +20,26 @@ import life.qbic.datamodel.dtos.projectmanagement.ProjectIdentifier
  *
  * @since 1.0.0
  */
-class CreateProject implements CreateProjectInput {
+class CreateProject implements CreateProjectInput, CreateProjectSpaceOutput {
 
     private final CreateProjectOutput output
     private final CreateProjectDataSource dataSource
 
+    private final CreateProjectSpace createProjectSpace
+    private ProjectApplication projectApplication
+
     private final Logging log = Logger.getLogger(CreateProject.class)
 
-    CreateProject(CreateProjectOutput output, CreateProjectDataSource dataSource) {
+    CreateProject(CreateProjectOutput output, CreateProjectDataSource dataSource, CreateProjectSpaceDataSource createProjectSpaceDataSource) {
```

Comment:
This will create a circular dependency

Reply:
It's not, I create the CreateProjectSpace use case in the constructor so there is no circular dependency.